db/orders.js

```diff
@@ -114,38 +117,38 @@ async function getHistory(userId) {
                         - join with line-items
                         - join with product
 */
-async function getAllOrders(orderId) {
+async function getOrderById(orderId) {
   try {
-    const { rows: allOrders } = await client.query(
+    const { rows: order } = await client.query(
```

Comment:
shouldn`t it be rows: [order]?

Reply:
No it shouldn't be because order is sent to the attachProductsToOrder function that takes an array.